platforms/android/tangram/src/main/cpp/platform_android.cpp

```diff
@@ -231,6 +233,21 @@ void AndroidPlatform::setContinuousRendering(bool _isContinuous) {
     jniEnv->CallVoidMethod(m_tangramInstance, setRenderModeMethodID, _isContinuous ? 1 : 0);
 }
 
+std::string AndroidPlatform::getAssetPath(const Tangram::Url& _path) const {
+
+    if (_path.scheme() == "asset") {
+        JniThreadBinding jniEnv(jvm);
+        auto assetPath = _path.path()[0] == '/' ? _path.path().substr(1) : _path.path();
```

Comment:
A _path_ is absolute if it starts with `/`, a _URL_ is absolute if it begins with a scheme. `Url::isAbsolute()` operates in this second sense. https://tools.ietf.org/html/rfc3986#section-4.3